### PR TITLE
Create Azimjon

### LINKED ISSUE #3: Bu Issue Azimga

### PR DESCRIPTION
**Pull request by Jovliyev**
Closes #3

### CODE REVIEW

File: PullRequesProject/Azimjon.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Jovliyev) [inline PullRequesProject/Azimjon.cs:5]: Islom bu coment senga

**Review by Jovliyev — COMMENTED** [review]
ewrteryhrjsrg

**Review by Elbekovich — APPROVED** [review]
azim aka

**Comment by Jovliyev** [comment]
xatoni tolipar

**Pull request merged**